appformer-kogito-bridge/src/main/java/org/appformer/kogito/bridge/client/context/impl/EditorContextProviderImpl.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.appformer.kogito.bridge.client.context.impl;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.appformer.kogito.bridge.client.context.EditorContextProvider;
+import org.appformer.kogito.bridge.client.context.interop.EditorContextWrapper;
+import org.appformer.kogito.bridge.client.interop.WindowRef;
+
+@ApplicationScoped
+public class EditorContextProviderImpl implements EditorContextProvider {
+
+    @Override
+    public Optional<String> getChannel() {
+        return nullSafe(EditorContextWrapper::getChannel);
+    }
+
+    @Override
+    public boolean isKogito() {
```

Comment:
@jesuino 
Hi. I think this should be removed/avoided. 
If "WindowRef.isGwtEditorBeansAvailable()" mean you have the `gwtEditorBeans` variable, this one may be not instantiated for a lot of different reasons - i.e. some error in code implementation/compilation, that are not related to the environment itself.

Reply:
Thanks for the feedback, @gitgabrio